cms/models.py

```diff
@@ -259,13 +259,9 @@ def get_context(self, request, *args, **kwargs):
             .select_related("thumbnail_image")
         )
 
-        featured_product = ProgramPage.objects.filter(
-            featured=True, program__live=True
-        ).select_related("program", "thumbnail_image").prefetch_related(
-            "program__courses__courseruns"
-        ) or CoursePage.objects.filter(
-            featured=True, course__live=True
-        )
+        featured_product = program_page_qset.filter(
```

Comment:
This change just reuses the `queryset` that we created above for Program and Course Pages, Plus we were not checking for `.live()` while obtaining the Featured Product.